projects/plugins/jetpack/extensions/blocks/videopress/v6/edit.js

```diff
@@ -1,18 +1,60 @@
 /**
  * WordPress dependencies
  */
-import { useBlockProps } from '@wordpress/block-editor';
+import { InspectorControls, useBlockProps } from '@wordpress/block-editor';
+import { PanelBody, ToggleControl, Tooltip } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
 import './editor.scss';
 
-export default function VideoPressEdit() {
+export default function VideoPressEdit( props ) {
+	const { attributes, setAttributes } = props;
+
+	const { controls } = attributes;
+
 	const blockProps = useBlockProps( {
 		className: 'wp-block-jetpack-videopress',
 	} );
 
-	return <div { ...blockProps }>{ __( 'VideoPress', 'jetpack' ) }</div>;
+	const renderControlLabelWithTooltip = ( label, tooltipText ) => {
+		return (
+			<Tooltip text={ tooltipText } position="top">
+				<span>{ label }</span>
+			</Tooltip>
+		);
+	};
+
+	const toggleAttribute = attribute => {
+		return newValue => {
+			setAttributes( { [ attribute ]: newValue } );
+		};
+	};
+
+	const blockSettings = (
+		<>
```

Comment:
minor: `<>` this wrapper is not needed for now

Reply:
Yeah I know, but we will need soon so I thought it's ok to leave it there